packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -682,6 +685,11 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
       builder: (BuildContext context, double currentSize, Widget? child) => LayoutBuilder(
         builder: (BuildContext context, BoxConstraints constraints) {
           _extent.availablePixels = widget.maxChildSize * constraints.biggest.height;
+
+          for (final _DraggableScrollableSheetScrollPosition position in _scrollController.positions) {
```

Comment:
This can be unnecessary if `availablePixels` didn't actually end up changing. Do we think it's worth optimizing that case out?